### DISCUSSION TITLE
New Thorium design

### DISCUSSION THREAD

**Discussion by moulciber** [Alex313031/thorium#439] [Show and tell]
Hi!

Your browser works great, but the only downside that caught my eye when i downloaded it was the **design**. I made a new icon for **Thorium** the other day, and i liked it so much that i'd like to suggest you use that as the **main Thorium design.**

![Thorium](https://github.com/Alex313031/thorium/assets/107422311/e62a36b7-ec2d-435d-8bd0-96de3d1f69b0)

**Comment by validMyID** [comment]
great logo....

ReactOS?

**Reply by moulciber** [reply]
No, I just took the concept, "radioactive elements."
